streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSegmentedBytesStore.java

```diff
@@ -124,8 +124,8 @@
 
         final List<S> searchSpace = keySchema.segmentsToSearch(segments, from, to, forward);
 
-        final Bytes binaryFrom = keySchema.lowerRange(keyFrom, from);
-        final Bytes binaryTo = keySchema.upperRange(keyTo, to);
+        final Bytes binaryFrom = keyFrom == null ? null : keySchema.lowerRange(keyFrom, from);
+        final Bytes binaryTo = keyTo == null ? null : keySchema.upperRange(keyTo, to);
```

Comment:
We already support `null` `binaryFrom` and `binaryTo` already.

Reply:
Do you have tests for this store? Generally can you make sure that your tests cover all the stores you added infinite range support.

Reply:
That's a good question. I'll check the test coverage later. Thanks.